tools/biobb_pytorch/biobb_apply_mdae.xml

```diff
@@ -0,0 +1,97 @@
+<tool id="biobb_pytorch_apply_mdae" name="ApplyMdae" version="@TOOL_VERSION@" profile="22.05">
+    <description>Apply a Molecular Dynamics AutoEncoder (MDAE) PyTorch model.</description>
+    <macros>
+      <token name="@TOOL_VERSION@">4.2.1</token>
+    </macros>
+
+    <requirements>
+      <requirement type="package" version="@TOOL_VERSION@">biobb_pytorch</requirement>
+    </requirements>
+
+    <command detect_errors="exit_code"><![CDATA[
+
+      `#Required inputs using file extension`
+      ln -s '$input_data_npy_path' ./input_data_npy_path.$input_data_npy_path.ext &&
+      ln -s '$input_model_pth_path' ./input_model_pth_path.pth &&
+
+      `#Optional inputs using file extension`
+      #if $config_json:
+        ln -s '$config_json' ./config_json.$config_json.ext &&
+      #end if
+
+      `#Required Outputs using file extension`
+      touch '$output_reconstructed_data_npy_path' &&
+      ln -s '$output_reconstructed_data_npy_path' ./output_reconstructed_data_npy_path.$output_reconstructed_data_npy_path.ext &&
+
+      `#Optional Outputs using file extension`
+      #if $output_latent_space_npy_path:
+        touch '$output_latent_space_npy_path' &&
+        ln -s '$output_latent_space_npy_path' ./output_latent_space_npy_path.$output_latent_space_npy_path.ext &&
+      #end if
+
+      `#Command`
+      apply_mdae
+
+      #if $config_json:
+        --config ./config_json.$config_json.ext
+      #end if
+
+      --input_data_npy_path ./input_data_npy_path.$input_data_npy_path.ext
+      --input_model_pth_path ./input_model_pth_path.pth
+      #if $output_latent_space_npy_path:
+      --output_latent_space_npy_path ./output_latent_space_npy_path.$output_latent_space_npy_path.ext
+      #end if
+      --output_reconstructed_data_npy_path ./output_reconstructed_data_npy_path.$output_reconstructed_data_npy_path.ext
+      ;
+      ]]>
```

Comment:
```suggestion
      ;
      mv ./output_reconstructed_data_npy_path.$output_reconstructed_data_npy_path.ext 'output_reconstructed_data_npy_path' &&
      #if $output_latent_space_npy_path:
           mv ./output_latent_space_npy_path.$output_latent_space_npy_path.ext 'output_latent_space_npy_path'
      #end if
      ]]>
```

Reply:
Thank you for your feedback. I removed the part you mentioned, used a literal in the command, and utilized the "from_work_dir" attribute. This has significantly simplified things.